packages/relay-group/CHANGELOG.md

```diff
@@ -0,0 +1,251 @@
+# @openzeppelin/defender-sdk-relay-client
```

Comment:
```suggestion
# @openzeppelin/defender-sdk-relay-group-client
```

Reply:
Pretty sure these are auto generated, so you could delete this since the changes doesn't apply from relay-client here.